include/deal.II/base/vectorization.h

```diff
@@ -165,16 +167,41 @@ class VectorizedArrayIterator
   VectorizedArrayIterator<T> &
   operator++()
   {
-    lane++;
+    AssertIndexRange(lane + 1, T::size() + 1) lane++;
```

Comment:
Not wrong but hard to read :-)
```suggestion
    AssertIndexRange(lane + 1, T::size() + 1);
    ++lane;
```